docs/configuration.md

```diff
@@ -1,4 +1,5 @@
 ---
+title: "Configuration"
```

Comment:
@rdblue the issue you were seeing with the search results container was not actually because it was behind the top navbar. It was caused by this `configuration.md` file not having a title set. This should fix it!